cypress/integration/samples.ts

```diff
@@ -93,17 +93,21 @@ context('Samples tests', () => {
       '[data-cy=sample-declaration-modal] [data-cy=save-and-continue-button]'
     ).click();
 
-    cy.get('[data-cy="questionaries-list-item"]').should('have.length', 1);
+    cy.finishedLoading();
+
+    cy.get('[data-cy="questionnaires-list-item"]').should('have.length', 1);
 
     cy.get('[data-cy="clone"]').click();
+
     cy.contains('OK').click();
-    cy.get('[data-cy="questionaries-list-item"]').should('have.length', 2);
+    cy.get('[data-cy="questionnaires-list-item"]').should('have.length', 2);
```

Comment:
Is this the right spelling of questionaries in the plural because when I search online there are so many different spellings 😅 
![Screenshot_43](https://user-images.githubusercontent.com/6333063/104704766-5d892500-5719-11eb-88d1-b624e1b68d77.png)